src/uct/ib/rdmacm/rdmacm_component.c

```diff
@@ -31,6 +31,12 @@ static ucs_config_field_t uct_rdmacm_cm_config_table[] = {
      ucs_offsetof(uct_rdmacm_cm_config_t, reserved_qpn),
                   UCS_CONFIG_TYPE_TERNARY},
 
+    {"REUSE_QPN", "no",
+     "Reuse reserved QPN for connection management:\n"
```

Comment:
@evgeny-leksikov 
What's about below document?
```
"Reuse reserved QPN for connection management when 1) RESERVED_QPN=on or 2) RESERVED_QPN=try and device support it:\n"
```

Reply:
> @evgeny-leksikov What's about below document?
> 
> ```
> "Reuse reserved QPN for connection management when 1) RESERVED_QPN=on or 2) RESERVED_QPN=try and device support it:\n"
> ```

what if `RESERVED_QPN=no` ?

Reply:
@evgeny-leksikov  If ```RESERVED_QPN=no```
1) When setting "REUSE_QPN=no"(default configuration), there's no error because it will use dummy QP
2) When setting "REUSE_QPN=yes", it'll report below error:
https://github.com/openucx/ucx/pull/8956/files#diff-a67fa46c1b19f20397f7a8fa05176ace0c5cd3af675a4132b5570b18359d10c2R185-R189
```
ucs_error("%s: reserved qpn is not supported, failed to use it", dev_name);
```

Reply:
Maybe `Reuse reserved QPN for connection management, the parameter can be enabled only if... `

Reply:
Done, it's
```
Reuse reserved QPN for connection management, the parameter can be enabled only if reserved qpn mode is used
```